pkg/cmd/diagnostics/diagnostics_list.go

```diff
@@ -0,0 +1,55 @@
+package diagnostics
+
+import (
+	"encoding/json"
+
+	"github.com/spf13/cobra"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-cli/pkg/cli"
+	"github.com/dustin/go-humanize"
+
+	diagnostics "github.com/dcos/dcos-core-cli/pkg/diagnostics/v2"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
```

Comment:
Do you use a specific rule for splitting imports here? Currently we only use 2 groups (one for stdlib and another one for the rest).

Reply:
I do though it looks like I broke it here. Normally I'll do 3 groups: stdlib, external imports, and imports within this project. I don't have any real reason for that besides it looking nicer to me so if you want it down to 2 groups, that's fine as well.

Reply:
In the future we could enforce a rule with goimports and have a CI check, in the meantime let's stick to 2 groups for consistency.